tests/test_fulltextsearch.py

```diff
@@ -0,0 +1,108 @@
+"""Tests for full_text_search.py"""
```

Comment:
Copy&paste copyright header "# Copyright 20XX Google Inc. ..." from other files, with year 2015. This declares that the copyright of this file belongs to Google.